demo/runners/agent_container.py

```diff
@@ -972,6 +971,25 @@ async def issue_credential(
 
             return cred_exchange
 
+        elif self.cred_type == CRED_FORMAT_VC_DI:
+            cred_preview = {
```

Comment:
not required, but we could refactor with section above, just set some variable and then have common code block